### DISCUSSION TITLE
raster tiles service down?

### DISCUSSION THREAD

**Discussion by yaniv14** [IsraelHikingMap/Site#2016] [General]
Hi,

is the raster tiles service is down?
[https://israelhiking.osm.org.il/Hebrew/Tiles/{z}/{x}/{y}.png](https://israelhiking.osm.org.il/Hebrew/Tiles/%7Bz%7D/%7Bx%7D/%7By%7D.png)

**Comment by HarelM** [comment]
Thanks for taking the time to report this issue.
We've just moved to a new server and this might be a side effect of this transition.
We'll look into it.
CC: @zstadler

**Comment by zstadler** [comment]
Indeed this is a result of the transition. The transition of the old raster tiles is still in progress. As a temporary workaround, the tiles are currently served by the old server.

**Reply by yaniv14** [reply]
Thank you
